### PR TITLE
fix http code 403 and 466 error for bilibili

### PR DESCRIPTION
**Pull request by ygmpkk**
Add referer header to fix http error code 403,
and user agent to fix http error code 466.

### CODE REVIEW

**Comment by ioilala** [comment]
please merge this commit and release a new youget, bilibili is not supported now.

**Comment by miseao** [comment]
please merge this commit and release a new you-get, bilibili is not supported now.

Please！！！！

**Comment by iawia002** [comment]
I have solved the problem of bilibili in [Lulu](https://github.com/iawia002/Lulu) (a you-get fork).

**Comment by ygmpkk** [comment]
@iawia002 这样做真的好吗，哈哈

**Comment by iawia002** [comment]
@ygmpkk 他实在不更新，我也没办法 😂，而且想再改改这个代码

**Comment by ayanamist** [comment]
lulu这个fork，不是在github上fork这个repo，而是自己重新上传一份，引入的这个fix也不保留原始commit信息，这样不好吧

**Comment by iawia002** [comment]
@ayanamist 我保留了原始信息的，你看下 commit 记录和 contributors 那些就知道了，只是我大改了它文件夹结构，所以看起来是没有原始信息

**Comment by lussier-q** [comment]
@iawia002 您好 非常感谢用了您的包我下载b站的问题解决了，但是想问下那个爱奇艺，优酷其他网站用youget也下不了的问题怎么解决呢，拜托了！

**Comment by iawia002** [comment]
@lussier-q lulu 现在爱奇艺，优酷也能下载，你用 lulu 遇到什么问题了吗

**Comment by xiiatuuo** [comment]
@iawia002 lulu为啥也是[DEBUG] HTTP Error with code466

**Comment by iawia002** [comment]
@xiiatuuo 你是说 lulu B 站下不了了？我试了一下没问题呀

**Comment by yangjie1863** [comment]
The file “bilibili.py” in you-get  0.4.1355 have been different from here

File: src/you_get/extractors/bilibili.py [hunk 1]
```diff
@@ -68,7 +68,7 @@ def api_req(self, cid, quality, bangumi, bangumi_movie=False, **kwargs):
             chksum = hashlib.md5(bytes(params_str+self.SEC2, 'utf8')).hexdigest()
             api_url = self.bangumi_api_url + params_str + '&sign=' + chksum
 
-        xml_str = get_content(api_url)
```
Comment (daohewang) [inline src/you_get/extractors/bilibili.py]: 此处有修改

**Comment by myheartorz** [comment]
The problem appears again, whether the AV or BV number is the same. I login again and clean the cache but still do not work

C:\Users\ASUS PC>you-get --debug https://www.bilibili.com/video/av800907262
[DEBUG] get_content: https://www.bilibili.com/video/av800907262
[DEBUG] HTTP Error with code403
[DEBUG] HTTP Error with code403
[DEBUG] HTTP Error with code403
you-get: version 0.4.1500, a tiny downloader that scrapes the web.
you-get: Namespace(URL=['https://www.bilibili.com/video/av800907262'], auto_rename=False, cookies=None, debug=True, extractor_proxy=None, force=False, format=None, help=False, http_proxy=None, info=False, input_file=None, insecure=False, itag=None, json=False, no_caption=False, no_merge=False, no_proxy=False, output_dir='.', output_filename=None, password=None, player=None, playlist=False, skip_existing_file_size_check=False, socks_proxy=None, stream=None, timeout=600, url=False, version=False)
Traceback (most recent call last):
  File "c:\users\asus pc\appdata\local\programs\python\python38\lib\runpy.py", line 194, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "c:\users\asus pc\appdata\local\programs\python\python38\lib\runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "C:\Users\ASUS PC\AppData\Local\Programs\Python\Python38\Scripts\you-get.exe\__main__.py", line 7, in <module>
  File "c:\users\asus pc\appdata\local\programs\python\python38\lib\site-packages\you_get\__main__.py", line 92, in main
    main(**kwargs)
  File "c:\users\asus pc\appdata\local\programs\python\python38\lib\site-packages\you_get\common.py", line 1798, in main
    script_main(any_download, any_download_playlist, **kwargs)
  File "c:\users\asus pc\appdata\local\programs\python\python38\lib\site-packages\you_get\common.py", line 1680, in script_main
    download_main(
  File "c:\users\asus pc\appdata\local\programs\python\python38\lib\site-packages\you_get\common.py", line 1327, in download_main
    download(url, **kwargs)
  File "c:\users\asus pc\appdata\local\programs\python\python38\lib\site-packages\you_get\common.py", line 1789, in any_download
    m.download(url, **kwargs)
  File "c:\users\asus pc\appdata\local\programs\python\python38\lib\site-packages\you_get\extractor.py", line 48, in download_by_url
    self.prepare(**kwargs)
  File "c:\users\asus pc\appdata\local\programs\python\python38\lib\site-packages\you_get\extractors\bilibili.py", line 183, in prepare
    initial_state = json.loads(initial_state_text)
  File "c:\users\asus pc\appdata\local\programs\python\python38\lib\json\__init__.py", line 341, in loads
    raise TypeError(f'the JSON object must be str, bytes or bytearray, '
TypeError: the JSON object must be str, bytes or bytearray, not NoneType

I don't know how to create new pull request. I'm sorry to feedback the problem in this way, I hope you can see it.

**Comment by myheartorz** [comment]
also missevan(猫耳FM) has problem

C:\Users\ASUS PC>you-get --debug https://www.missevan.com/sound/player?id=575494
[DEBUG] get_content: https://www.missevan.com/sound/getsound?soundid=575494
[DEBUG] get_content: https://www.missevan.com/sound/getdm?soundid=575494
you-get: version 0.4.1500, a tiny downloader that scrapes the web.
you-get: Namespace(URL=['https://www.missevan.com/sound/player?id=575494'], auto_rename=False, cookies=None, debug=True, extractor_proxy=None, force=False, format=None, help=False, http_proxy=None, info=False, input_file=None, insecure=False, itag=None, json=False, no_caption=False, no_merge=False, no_proxy=False, output_dir='.', output_filename=None, password=None, player=None, playlist=False, skip_existing_file_size_check=False, socks_proxy=None, stream=None, timeout=600, url=False, version=False)
Traceback (most recent call last):
  File "c:\users\asus pc\appdata\local\programs\python\python38\lib\runpy.py", line 194, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "c:\users\asus pc\appdata\local\programs\python\python38\lib\runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "C:\Users\ASUS PC\AppData\Local\Programs\Python\Python38\Scripts\you-get.exe\__main__.py", line 7, in <module>
  File "c:\users\asus pc\appdata\local\programs\python\python38\lib\site-packages\you_get\__main__.py", line 92, in main
    main(**kwargs)
  File "c:\users\asus pc\appdata\local\programs\python\python38\lib\site-packages\you_get\common.py", line 1798, in main
    script_main(any_download, any_download_playlist, **kwargs)
  File "c:\users\asus pc\appdata\local\programs\python\python38\lib\site-packages\you_get\common.py", line 1680, in script_main
    download_main(
  File "c:\users\asus pc\appdata\local\programs\python\python38\lib\site-packages\you_get\common.py", line 1327, in download_main
    download(url, **kwargs)
  File "c:\users\asus pc\appdata\local\programs\python\python38\lib\site-packages\you_get\common.py", line 1789, in any_download
    m.download(url, **kwargs)
  File "c:\users\asus pc\appdata\local\programs\python\python38\lib\site-packages\you_get\extractors\missevan.py", line 302, in download_by_url
    super().download_by_url(url, **kwargs)
  File "c:\users\asus pc\appdata\local\programs\python\python38\lib\site-packages\you_get\extractor.py", line 48, in download_by_url
    self.prepare(**kwargs)
  File "c:\users\asus pc\appdata\local\programs\python\python38\lib\site-packages\you_get\extractors\missevan.py", line 222, in prepare
    self.__prepare_dispatcher.dispatch(self.url, self, **kwargs)
  File "c:\users\asus pc\appdata\local\programs\python\python38\lib\site-packages\you_get\extractors\missevan.py", line 73, in dispatch
    return fun(*args, **kwargs)
  File "c:\users\asus pc\appdata\local\programs\python\python38\lib\site-packages\you_get\extractors\missevan.py", line 197, in prepare_sound
    self.streams = self.setup_streams(sound)
  File "c:\users\asus pc\appdata\local\programs\python\python38\lib\site-packages\you_get\extractors\missevan.py", line 204, in setup_streams
    uri = _get_resource_uri(sound, stream_type)
  File "c:\users\asus pc\appdata\local\programs\python\python38\lib\site-packages\you_get\extractors\missevan.py", line 92, in _get_resource_uri
    uri = data[stream_type['url_json_key']]
KeyError: 'soundurl_64'

**Pull request merged**